packages/workflow/src/RoutingNode.ts

```diff
@@ -209,17 +211,29 @@ export class RoutingNode {
 
 				returnData.push(...responseData);
 			} catch (error) {
-				if (get(this.node, 'continueOnFail', false)) {
-					returnData.push({ json: {}, error: error.message });
+				interface RoutingError extends NodeError {
```

Comment:
maybe `RoutingError` should be called `AxiosError`, so that whenever we upgrade to axios 1.x, we can switch checking `if (error instance of AxiosError)`.